includes/admin/class-wc-payments-admin.php

```diff
@@ -37,7 +40,7 @@ public function __construct( WC_Payment_Gateway_WCPay $gateway, WC_Payments_Acco
 		$this->account       = $account;
 
 		// Add menu items.
-		add_action( 'admin_menu', [ $this, 'add_payments_menu' ], 9 );
+		add_action( 'admin_menu', [ $this, 'add_payments_menu' ], PHP_INT_MAX );
```

Comment:
Was there a reason this was previously set at `9`?  This is set to `PHP_INT_MAX` to detect any other payment items that may be added to the payment category so we can move WC Pay into its own menu if that happens.

Reply:
This change is no longer needed with the new plugin structure.